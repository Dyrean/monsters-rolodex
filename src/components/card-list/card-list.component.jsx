import "./card-list.styles.css";
import React from 'react';
import { Card } from '../card/card.component';

export const CardList = (props) => {
  return (
    <div className='card-list'>
      {props.monsters.map((monster) => (
        <Card key={monster.id} monster={monster}/>
      ))}
    </div>
  );
};
