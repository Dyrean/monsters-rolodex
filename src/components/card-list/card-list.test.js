import { shallow } from "enzyme";
import { CardList } from './card-list.component'
import React from "react";

test("Card component...", () => {
    const mockMonsters = [{
        id: 1,
        name: 'Testing',
        username: 'John',
        email: 'john@gmail.com'
    }]
  expect(shallow(<CardList monsters={mockMonsters} />)).toMatchSnapshot();
});
