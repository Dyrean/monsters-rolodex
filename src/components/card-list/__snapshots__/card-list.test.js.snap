// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component... 1`] = `ShallowWrapper {}`;
